### ISSUE TITLE
Setting master volume level with percentages

### ISSUE THREAD

**Issue by WoolDoughnut310** [AndreMiras/pycaw#13]
I have this [file](https://github.com/AndreMiras/pycaw/files/2748128/volume.zip) containing the script i came up with. But I was wondering if instead of decibels to set the volume it can use real percentages as most thinngs have volume that goes up to 100%.

**Comment by AndreMiras** [comment]
Yes I think that's a valid concern, even though so far pycaw is more focusing on making a dumb Python interface to the Windows core audio library.
Feel free to give it a try to pull request something on top for changing the volume in percentages :+1:

**Comment by csevast** [comment]
Aren't the 'GetChannelVolumeLevelScalar' and 'SetMasterVolumeLevelScalar' methods of the 'IAudioEndpointVolume' class suitable? I'm using them to connect a slider controls in a GUI with the master volume of the OS.

**Comment by AndreMiras** [comment]
Yes I forgot about it, they seem to be from the documentation, thanks for the head up @csevast 
So I guess we can close that ticket @WoolDoughnut310 right?

**Comment by WoolDoughnut310** [comment]
@AndreMiras I have fixed that now and will create a pull request of how I changed the script to work with percentages.

**Comment by realJosef** [comment]
> Actually here is a zip with my modified script.
> 
> [volume.zip](https://github.com/AndreMiras/pycaw/files/2753497/volume.zip)

i've tryed using yours, but cant get it running.
what input typed do the functions need? the Device name fir mute/unmute

**Comment by rjargumido** [comment]
Here is my implementation of pycaw of controlling master volume level using percentages.
[mastersoundcontrol.zip](https://github.com/AndreMiras/pycaw/files/5358552/mastersoundcontrol.zip)
